server/db/schema.models.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+const dailyEntriesName = 'daily_entries';
+
```

Comment:
what's the status / plan for the users and goals models?

Reply:
That should be done or decided by the owners of the Authentication and users features. Essentially all we care about is the user_id String, which can be anything